web/api/v1/api_test.go

```diff
@@ -1222,6 +1223,7 @@ func (f *fakeDB) Dir() string {
 	return dir
 }
 func (f *fakeDB) Snapshot(dir string, withHead bool) error { return f.err }
+func (f *fakeDB) Head() *tsdb.Head                         { return &tsdb.Head{} }
```

Comment:
It'd be great to see some real test data here

Reply:
Thank you, Dustin.
I am working on it :)